db/R/get.trait.data.R

```diff
@@ -82,8 +82,13 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
   trait.data.check <- query.traits(spstr, traits, con = dbcon, update.check.only=TRUE)
   traits <- names(trait.data.check)
 
+  # Set forceupdate FALSE if it's a string (backwards compatible with 'AUTO' flag used in the past)
+  if(!is.logical(forceupdate)) {
+    forceupdate <- FALSE
+  }
+  
   # check to see if we need to update
-  if (forceupdate || !as.logical(forceupdate)) {
+  if (!forceupdate) {
```

Comment:
@rykelly question here. If the logic was previously that we run this when update = TRUE or AUTO, and we've deprecated the update = FALSE option (now you just drop the meta-analysis block), wouldn't that imply that this check should be whether the meta.analysis tag exists? Is it not run in both cases?